### PR TITLE
[CBRD-23118] fix fetching pages after log reset

### PR DESCRIPTION
**Pull request by ribram**
http://jira.cubrid.org/browse/CBRD-23118

Disable fetching archive pages from active log after it was reset.

This is enough to fix vacuum data issue of partial restore.

### CODE REVIEW

File: src/transaction/log_page_buffer.c [hunk 1]
```diff
@@ -1901,9 +1901,12 @@ logpb_read_page_from_file (THREAD_ENTRY * thread_p, LOG_PAGEID pageid, LOG_CS_AC
       assert (LOG_CS_OWN (thread_p));
     }
 
-  if (logpb_is_page_in_archive (pageid)
-      && (LOG_ISRESTARTED () == false || (pageid + LOGPB_ACTIVE_NPAGES) <= log_Gl.hdr.append_lsa.pageid))
+  // some archived pages may be still in active log; check if they can be fetched from active.
+  bool is_archive_page_in_active_log = (pageid + LOGPB_ACTIVE_NPAGES) > log_Gl.hdr.append_lsa.pageid;
```
Comment (alexstanro) [inline src/transaction/log_page_buffer.c]: This 2 flags have meaning if logpb_is_page_in_archive is true. So, they should be computed after calling logpb_is_page_in_archive.

Comment (ribram) [reply src/transaction/log_page_buffer.c]: fixed

**Comment by ribram** [comment]
@eseokoh 
the different timestamp resulted from failure of test case. please find my comment on jira issue.

**Comment by ribram** [comment]
@eseokoh I think there are no other issues. This may be reviewed/merged.

**Pull request merged**